docs/user-guide/petset.md

```diff
@@ -392,7 +392,8 @@ $ grace=$(kubectl get po web-0 --template '{{.spec.terminationGracePeriodSeconds
 $ kubectl delete petset,po -l app=nginx
 $ sleep $grace
 $ kubectl delete pvc -l app=nginx
-{% endraw %}```
+{% endraw %}
+```
```

Comment:
this part has always been weird, looks like we need to break gh rendering to get pages to do it right. I played around a bit with the formatting and keeping it like it is seemed like the best option. What you currently have leaks formatting over into the Troubleshooting section, vs https://github.com/kubernetes/kubernetes.github.io/blob/master/docs/user-guide/petset.md#deleting-a-petset which is a little less bad.

Any reason to move it to a newline (or suggesting to make it work uniformly)?